central/nodecomponentedge/datastore/singleton.go

```diff
@@ -16,10 +20,18 @@ var (
 )
 
 func initialize() {
-	storage := dackbox.New(globaldb.GetGlobalDackBox())
-	searcher := search.New(storage, index.New(globalindex.GetGlobalIndex()))
+	var storage store.Store
+	var searcher search.Searcher
 
-	ad = New(globaldb.GetGlobalDackBox(), storage, index.New(globalindex.GetGlobalIndex()), searcher)
+	if features.PostgresDatastore.Enabled() {
+		storage = postgres.New(globaldb.GetPostgres())
+		searcher = search.New(storage, index.New(globalindex.GetGlobalIndex()))
```

Comment:
This should be based on postgres and not on bleve

Reply:
😱